src/base.js

```diff
@@ -4,14 +4,16 @@ const Augur = require("augurbot"),
   path = require("path"),
   u = require("./utils/utils");
 
+const disabledCommands = ["twitch.js"];
```

Comment:
I should test it with twitch enabled, didn't get an API key so I disabled it initially